### ISSUE TITLE
lighthouse documentation is not working

### ISSUE THREAD

**Issue by lbrydocs** [lbryio/lighthouse#47]
Try the autocomplete function here, it fails:

https://lbryio.github.io/lighthouse/

**Comment by tiger5226** [comment]
The autocomplete query suffers from the same problems that search did previously. the value section of the elastic document is of type nested, which means the query needs to also be a nested query. The result therefore is actually only searching the name of a claim instead of search for the best auto complete term across the main fields of title, description and author. Below is an example that is current:

https://lighthouse.lbry.io/autocomplete?s=test%20a

Result:
```
["make-a-test-tube-thunderstorm","Make a Test Tube Thunderstorm!","NurdRage","a-test-of-wills-charles-todd-mobi","A Test of Wills By Charles Todd Mobi Format","upload-test-12-11-17-a","spee.ch","how-to-detect-a-secret-nuclear-test","How To Detect A Secret Nuclear Test","minutephysics"]
```

Now if you look at the first result `make-a-test-tube-thunderstorm` and search that with 

https://lighthouse.lbry.io/search?s=test%20a

Result:
```
[{"name":"test","claimId":"a607349ce83d3a86bac87a967ce7f9647e1ba736","value":{"claimType":"streamType","stream":{"metadata":{"preview":"","license":"Creative Commons Attribution 4.0 International","licenseUrl":"","thumbnail":"","nsfw":false,"author":"test","description":"test","language":"en","title":"test","version":"_0_1_0"},"source":{"sourceType":"lbry_sd_hash","source":"29ad218c61c599499b22c17228371b5fe9a6e725edc9ef691a7819b3e7406500467852920629fbcddbcacf13d31579d4","version":"_0_0_1","contentType":"image/png"},"version":"_0_0_1"},"version":"_0_0_1"}},{"name":"make-a-test-tube-thunderstorm","claimId":"40d193673b0730907449a3dde387b2cdb0314eff",
...
```
You can see that the claim name `test` is first but then `"name":"make-a-test-tube-thunderstorm"` is second. So it is only searching the name field. 

The internal server error should be a separate issue. We should not hit an error by entering a query. I will create another issue for this.

**Comment by tiger5226** [comment]
I split out the original issue into 3 separate problems that need to be solved.

**Comment by filipnyquist** [comment]
This is now solved by PR #60. Closing!
